test/perf-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+require('babel-register');
+const Benchmark = require('benchmark');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+
+const filename = path.resolve(__dirname, 'screenshotter/ss_data.yaml');
+const data = yaml.load(fs.readFileSync(filename, 'utf-8'));
+
+console.log('compiling katex...');
+const katex = require('../katex');
+console.log('');
+
+const suite = new Benchmark.Suite;
```

Comment:
Nits: This line seems like it is worth a comment for readers who don't already know about this benchmark package. I skimmed over it the first time and then had to come back to it. Also, I think that the variable name 'tests' would make the code below a little clearer.